cmd/certsuite/run/run.go

```diff
@@ -39,8 +39,8 @@ func NewCommand() *cobra.Command {
 	runCmd.PersistentFlags().Bool("include-web-files", false, "Save web files in the configured output folder")
 	runCmd.PersistentFlags().Bool("enable-data-collection", false, "Allow sending test results to an external data collector")
 	runCmd.PersistentFlags().Bool("create-xml-junit-file", false, "Create a JUnit file with the test results")
-	runCmd.PersistentFlags().String("tnf-image-repository", "quay.io/redhat-best-practices-for-k8s", "The repository where TNF images are stored")
-	runCmd.PersistentFlags().String("tnf-debug-image", "certsuite-probe:v0.0.7", "Name of the certsuite-probe image")
+	runCmd.PersistentFlags().String("certsuite-image-repository", "quay.io/redhat-best-practices-for-k8s", "The repository where Certsuite images are stored")
```

Comment:
This is non-backward compatible change. @sebrandon1 , probably DCI, and QE will need to update to use these new flag names. Also the next certsuite release notes should reflect this changes.

Reply:
Please tell us what are the variables we need to change, to adapt our code to this new change. Thanks!